### ISSUE TITLE
Formatting issue with C.3.5.4

### ISSUE THREAD

**Issue by lsherida** [risk-redux/typist#27]
The `C.3.5.4 - IT infrastructure maintenance` entry has a paragraph tag misplaced in the `lead` text.

**Comment by egyptiankarim** [comment]
Yeah... This is a result of a pretty messy design decision on my part to parse out the first sentence as the `lead` by searching for a '.' string literal... It's not ideal, obviously, but the best I can do for now (short of actually manually dropping markers into all of the text I've parsed out of the canonical NIST PDFs) is:

- Parse out '. ' instead, which makes things a tiny bit neater.
- Drop the notion of `lead` text all together.

@lsherida you're probably one of the people that actually uses this tool, so what's your preference?
